docs/1-creating-a-graphql-server-project.md

```diff
@@ -153,71 +151,48 @@ Commands Explained
     });
     ```
 
-    > Your Startup.cs should now look like the following:
+   > Your Startup.cs should now look like the following:
 
     ```csharp
-    using System;
-    using System.Collections.Generic;
-    using System.Linq;
-    using System.Threading.Tasks;
-    using ConferencePlanner.GraphQL;
-    using ConferencePlanner.GraphQL.Data;
-    using Microsoft.AspNetCore.Builder;
-    using Microsoft.AspNetCore.Hosting;
-    using Microsoft.AspNetCore.Http;
+    using ConferencePlanner;
+    using ConferencePlanner.Data;
     using Microsoft.EntityFrameworkCore;
-    using Microsoft.Extensions.DependencyInjection;
-    using Microsoft.Extensions.Hosting;
-
-    namespace GraphQL
-    {
-        public class Startup
-        {
-            // This method gets called by the runtime. Use this method to add services to the container.
-            // For more information on how to configure your application, visit https://go.microsoft.com/fwlink/?LinkID=398940
-            public void ConfigureServices(IServiceCollection services)
-            {
-                services.AddDbContext<ApplicationDbContext>(
-                    options => options.UseSqlite("Data Source=conferences.db"));
 
-                services
-                    .AddGraphQLServer()
-                    .AddQueryType<Query>();
-            }
 
-            // This method gets called by the runtime. Use this method to configure the HTTP request pipeline.
-            public void Configure(IApplicationBuilder app, IWebHostEnvironment env)
-            {
-                if (env.IsDevelopment())
-                {
-                    app.UseDeveloperExceptionPage();
-                }
+    var builder = WebApplication.CreateBuilder(args);
+    
+    builder.Services.AddDbContext<ApplicationDbContext>(
+        options => options.UseSqlite("Data Source=conferences.db"));
+    builder.Services
+            .AddGraphQLServer()
+            .AddQueryType<RootQuery>()
+            .AddMutationType<Mutation>();
+    
+    var app = builder.Build();
+    app.UseRouting();
```

Comment:
We no longer need the routing stuff here ....

app.MapGraphQL() is all that is needed.